### ISSUE TITLE
Check type of OpSampledImage image component vs result type's image component

### ISSUE THREAD

**Issue by dneto0** [KhronosGroup/SPIRV-Tools#838]
See test case in https://github.com/KhronosGroup/glslang/issues/1073

**Comment by johnkslang** [comment]
Since it is not yet entirely clear the spec should not be changed here, can this one be prioritized behind missing validator pieces that are unquestionably needed but have been missing for some time now?

**Comment by dneto0** [comment]
Agreed. Deprioritized to the lowest level, and added a "Maybe" in the title.
